src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -2118,4 +2118,24 @@ public void setActivePushToApplication(PushToApplication application, PushToAppl
             manager.updateApplicationAction();
         }
     }
+
+    public NewLineSeparator getNewLineSeparator() {
+        String newline = get(JabRefPreferences.NEWLINE);
```

Comment:
we have a  method in the OS class which already returns the system line separator

Reply:
Thanks for looking into the code, I just realized another mistake I made.
In this mentioned case: OS.NEWLINE returns the newline-seperator of the user system. This method here concerns the stored newline-seperator in the preferences, the user can change just as she likes.

Reply:
Took me a moment, now I think I understand your concern: The variable OS.NEWLINE, which normally returns the system-newline-separator is being overwritten by the method setNewLineSeparator. My question: Is this not a bug? Should the OS.NEWLINE return the user-system newLineSeparator or the one stored in the preferences? I'm a little bit confused.

Reply:
I think the code is correct the way it is right now. The `OS.NEWLINE` returns the os-default newline character except if the user chooses to overwrite it in the preferences. It's a bit confusing (especially since `OS.NEWLINE` suggests that it is a constant).